src/apps/frontend/pages/authentication/otp/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+import constants from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import OtpForm from './otp-form';
+
+export const Otp: React.FC = () => {
+  const navigate = useNavigate();
+  const onSuccess = () => {
+    toast.success('OTP verification success');
```

Comment:
These are quite poor messages. I understand that copyright does not fall under engineering strictly but these are simple enough things we can Google, leverage Generate AI and others to do better. Similarly, other places, where we display user facing messages